coralproject/pillar
# Env add

## What does this PR do?

Adds a sample env.list

## How do I test this PR?

docker build .
docker run --env-file ./env.list --publish 8080:8080 <container id>
(depends on mongo and xenia)

Verify pillar starts up without any missing environment variables.

@coralproject/backend
